sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -241,8 +241,8 @@ def dup_topx(state, arg):
   state.stack += state[-arg:]
 
 
-store_attr = pop_top
-delete_attr = nop
```

Comment:
Can you provide more context on this change? Where there any previous issues/failures?